build.gradle

```diff
@@ -337,6 +340,9 @@ project('controller') {
     applicationName = "pravega-controller"
     mainClassName = "io.pravega.controller.server.Main"
     applicationDefaultJvmArgs = ["-server", "-Xms128m", "-XX:+HeapDumpOnOutOfMemoryError",
+                                 "-XX:+PrintGCDetails", "-XX:+PrintGCDateStamps",
+                                 "-Xloggc:PRAVEGA_APP_HOME/logs/gc.log", "-XX:+UseGCLogFileRotation",
```

Comment:
This path seems same for both controller and segmentstore, I guess not an issue with multi-node or when deployed on containers. Please review this for singlenode and rename if necessary.